src/domains/user/utils/recommendation.ts

```
@@ -0,0 +1,72 @@
+export const translateObjectValuesEngToKr = (recommendationStep2: RecommendationStep2ResultType) =>
+  Object.keys(recommendationStep2).reduce(
+    (result, key) => ({
+      ...result,
+      [key]: recommendationStep2[key as keyof RecommendationStep2Type].map((ele) =>
+        recommendationDictionary.translate(ele)
+      )
+    }),
+    {} as RecommendationStep2Type
+  );
+
+export const translateObjectValuesKrToEng = (recommendationStep2: RecommendationStep2Type) =>
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)